nuget/spec/dependabot/nuget/update_checker/compatibility_checker_spec.rb

```diff
@@ -74,41 +74,109 @@
   context "#compatible?" do
     subject(:compatible) { checker.compatible?(version) }
 
-    context "when the `.nuspec` has groups without a `targetFramework` attribute" do
-      let(:version) { "5.0.3" }
+    before do
+      stub_request(:get, "https://api.nuget.org/v3/registration5-gz-semver2/microsoft.appcenter.crashes/index.json")
```

Comment:
The test diff looks worse than it really is.  This section got moved up a level because it's common to both `context`s below it, otherwise it's a new test case.